src/app/helpers/movimientos/movimientosHelper.ts

```diff
@@ -68,6 +68,14 @@ export class MovimientosHelper {
      * @param nombre nombre del formato
      * @returns  <Observable> data of the object registered at the DB. undefined if the request has errors
      */
+
+   /**
+     * Trae el formato movimiento según el nombre
+     * If the response has errors in the OAS API it should show a popup message with an error.
+     * If the response is successs, it returns the object's data.
+     * @param nombre nombre del formato
```

Comment:
Esta documentación es en [formato javadoc](https://es.wikipedia.org/wiki/Javadoc), VSCode la detecta automáticamente y va ayudando mientras se programa si está bien estructurada esta documentación. En este caso el `@param` no se llama `nombre`, cambiar por `query` que es el que está
```suggestion
     * @param query nombre del formato
```
Adicionalmente, ajustar/quitar los _"If the response ..."_